bot/exts/filters/webhook_remover.py

```diff
@@ -44,9 +44,12 @@ async def delete_and_respond(self, msg: Message, redacted_url: str) -> None:
             return
 
         await msg.channel.send(ALERT_MESSAGE_TEMPLATE.format(user=msg.author.mention))
-
+        if webhook_deleted:
+            delete_state = "The webhook was successfully deleted."
+        else:
+            delete_state = "There was an error when deleting the webhook, it might have already been removed."
         message = (
-            f"{format_user(msg.author)} posted a Discord webhook URL to {msg.channel.mention}. "
+            f"{format_user(msg.author)} posted a Discord webhook URL to {msg.channel.mention}.{delete_state} "
```

Comment:
```suggestion
            f"{format_user(msg.author)} posted a Discord webhook URL to {msg.channel.mention}. {delete_state} "
```
You don't have a space in `delete_state` so we need to add it here